Events/showImage.m

```diff
@@ -137,34 +137,18 @@
     q(1).name = 'event Name';
     q(1).sort = 'edit';
     q(1).data = 'Draw Image';
+        
+    q(2).name = 'Behaviors';
```

Comment:
It was already in the code. It doesn't take any input, only clarifies the following options.